### PR TITLE
[Tagged: Capabilities articles] Linkify and check all hashtags

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Linkify all hashtags in Capabilities articles.

### CODE REVIEW

File: src/site/content/en/blog/badging-api/index.md [hunk 1]
```diff
@@ -210,8 +210,9 @@ Planning to use the App Badging API on your site? Your public support helps the
 Chrome team to prioritize features, and shows other browser vendors how critical
 it is to support them.
 
-* Send a Tweet to [@ChromiumDev](https://twitter.com/chromiumdev) tagged with
-  `#badgingapi` and let us know where and how you're using it.
+* Send a tweet to [@ChromiumDev](https://twitter.com/chromiumdev) tagged with the hashtag
```
Comment (jpmedley) [inline src/site/content/en/blog/badging-api/index.md:213]:
```suggestion
* Send a tweet to [@ChromiumDev](https://twitter.com/chromiumdev) using the hashtag
```

File: src/site/content/en/blog/file-system-access/index.md [hunk 2]
```diff
@@ -584,8 +584,9 @@ Planning to use the File System Access API on your site? Your public support hel
 features, and shows other browser vendors how critical it is to support them.
 
 - Share how you plan to use it on the [WICG Discourse thread][wicg-discourse]
-- Send a Tweet to [@ChromiumDev][cr-dev-twitter] with `#filesystemaccess` and let us know where and how
-  you're using it.
+- Send a tweet to [@ChromiumDev][cr-dev-twitter] with hashtag
```
Comment (jpmedley) [inline src/site/content/en/blog/file-system-access/index.md:587]:
```suggestion
- Send a tweet to [@ChromiumDev][cr-dev-twitter] with the hashtag
```

**Review by jpmedley — CHANGES_REQUESTED** [review]
Editing the same thing in 20 places is the kind of thing that makes me want to gnaw my own leg off. Thank you.

File: src/site/content/en/blog/file-system-access/index.md (hunk 2 above)
Comment (tomayac) [reply src/site/content/en/blog/file-system-access/index.md]: Made this call to action consistent in https://github.com/GoogleChrome/web.dev/pull/4722/commits/a2da5b8fc1d23310762c809a007b709e24c1a909.

File: src/site/content/en/blog/badging-api/index.md (hunk 1 above)
Comment (tomayac) [reply src/site/content/en/blog/badging-api/index.md]: Made this call to action consistent in https://github.com/GoogleChrome/web.dev/pull/4722/commits/a2da5b8fc1d23310762c809a007b709e24c1a909.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**